assets/src/js/media_archive_editor_view.js

```diff
@@ -0,0 +1,65 @@
+import {createRoot} from '@wordpress/element';
+import ArchivePicker from '../js/Components/ArchivePicker/ArchivePicker';
+
+const frame = wp.media.view.MediaFrame.Select;
+
+(function() {
+  wp.media.view.MediaFrame.Select = frame.extend({
+    initialize() {
+      frame.prototype.initialize.apply(this, arguments);
+
+      const State = wp.media.controller.State.extend({
+        insert() {
+          this.frame.close();
+        },
+      });
+
+      this.states.add([
+        new State({
+          id: 'mediaArchive',
+          search: false,
+          title: 'Media Archive',
+        }),
+      ]);
+
+      this.on('content:render:mediaArchive1', this.renderMediaArchive, this);
```

Comment:
Is that intentional the `mediaArchive1` ?

Reply:
No that's an error, should say `mediaArchive` instead, fixing now.